concent_api/utils/helpers.py

```diff
@@ -159,3 +159,10 @@ def calculate_maximum_download_time(size: int) -> int:
             settings.MINIMUM_UPLOAD_RATE,
         ).total_seconds()
     )
+
+
+def join_messages(*msg):
+    list_of_messages = []
+    for msg_part in msg:
+        list_of_messages.append(msg_part.strip())
+    return ' '.join(m for m in list_of_messages if m not in ['', None])
```

Comment:
This could be shortened to:

``` python
def join_messages(*messages):
    return ' '.join(m.strip() for m in messages if m not in ['', None])
```

Reply:
`.strip()` is just a safety so there can't be more then one whitespace, but okey.